test/src/test/java/org/neo4j/ogm/persistence/examples/cineasts/annotated/CineastsRelationshipEntityTest.java

```diff
@@ -907,9 +908,9 @@ public void testFilterOnRelationshipEntity() throws Exception {
         ootfRating2.setMovie(ootf);
         ootfRating2.setUser(otto);
 
-        pulpFiction.setRatings(newHashSet(pulpRating, pulpRating2));
-        ootf.setRatings(newHashSet(ootfRating, ootfRating2));
-        otto.setRatings(newHashSet(pulpRating2, ootfRating2));
+        pulpFiction.setRatings(new HashSet<>(Arrays.asList(pulpRating, pulpRating2)));
```

Comment:
Hehe I know it is bike shedding but this is is unnecessary ugliness of highest order :-) Having 2.5 mb *test* dependency is totally worth it.

Reply:
I'd go Java 10 in a sec if anyone lets me… `HashSet.of()` and done. :)